DatadogFlags/Sources/Models/AnyValue.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+public enum AnyValue: Equatable {
```

Comment:
I just realized we could use `AnyCodable` instead. Is it OK to use `DatadogInternal` types in public APIs?

Reply:
I discussed this offline with @ncreated and we decided to leave `AnyValue` for our use case. We think that the type specification it provides is better suited for public APIs.